lib/server.go

```diff
@@ -490,6 +483,36 @@ func NewServer(
 	_mempool := NewDeSoMempool(_chain, _rateLimitFeerateNanosPerKB,
 		_minFeeRateNanosPerKB, _blockCypherAPIKey, _runReadOnlyUtxoViewUpdater, _dataDir,
 		_mempoolDumpDir, false)
+	_posMempool := NewPosMempool()
+
+	// Initialize the PoS mempool. We need to initialize a best-effort UtxoView based on the current
+	// known state of the chain. This will all be overwritten as we process blocks later on.
+	currentUtxoView, err := _chain.GetUncommittedTipView()
+	if err != nil {
+		return nil, errors.Wrapf(err, "NewServer: Problem initializing latest UtxoView"), true
+	}
+	currentGlobalParamsEntry := currentUtxoView.GetCurrentGlobalParamsEntry()
+	latestBlockHash := _chain.blockTip().Hash
+	latestBlock := _chain.GetBlock(latestBlockHash)
```

Comment:
I think @diamondhands0 actually wants us to load up the last 60 blocks. We can configure the # of blocks via some param, but we're going to need to make it resilient to hypersync not syncing full blocks. So if we can only get the last 10 blocks instead of 60, that's okay too. I'm okay punting these changes to another PR.

Reply:
Sounds good. Good callout. I'll add a todo for myself for this. For now, I just want this wired up with the simplest params so we can regtest. We can parametrize in another PR>